Source/JavaScriptCore/inspector/protocol/ApplicationCache.json

```diff
@@ -1,92 +0,0 @@
-            "properties": [
-                { "name": "frameId", "$ref": "Network.FrameId", "description": "Frame identifier." },
-                { "name": "manifestURL", "type": "string", "description": "Manifest URL." },
-                { "name": "status", "type": "integer", "description": "Application cache status." }
-            ]
-        }
-    ],
-    "commands": [
-        {
-            "name": "getFramesWithManifests",
-            "description": "Returns array of frame identifiers with manifest urls for each frame containing a document associated with some application cache.",
-            "returns": [
-                { "name": "frameIds", "type": "array", "items": { "$ref": "FrameWithManifest" }, "description": "Array of frame identifiers with manifest urls for each frame containing a document associated with some application cache." }
-            ]
-        },
-        {
-            "name": "enable",
-            "description": "Enables application cache domain notifications."
-        },
-        {
-            "name": "disable",
-            "description": "Disable application cache domain notifications."
-        },
-        {
-            "name": "getManifestForFrame",
-            "description": "Returns manifest URL for document in the given frame.",
-            "parameters": [
-                { "name": "frameId", "$ref": "Network.FrameId", "description": "Identifier of the frame containing document whose manifest is retrieved." }
-            ],
-            "returns": [
-                { "name": "manifestURL", "type": "string", "description": "Manifest URL for document in the given frame." }
-            ]
-        },
-        {
-            "name": "getApplicationCacheForFrame",
-            "description": "Returns relevant application cache data for the document in given frame.",
-            "parameters": [
-                { "name": "frameId", "$ref": "Network.FrameId", "description": "Identifier of the frame containing document whose application cache is retrieved." }
-            ],
-            "returns": [
-                { "name": "applicationCache", "$ref": "ApplicationCache", "description": "Relevant application cache data for the document in given frame." }
-            ]
-        }
-    ],
-    "events": [
-        {
-            "name": "applicationCacheStatusUpdated",
-            "parameters": [
-                { "name": "frameId", "$ref": "Network.FrameId", "description": "Identifier of the frame containing document whose application cache updated status." },
-                { "name": "manifestURL", "type": "string", "description": "Manifest URL." },
-                { "name": "status", "type": "integer", "description": "Updated application cache status." }
-            ]
-        },
-        {
-            "name": "networkStateUpdated",
-            "parameters": [
-                { "name": "isNowOnline", "type": "boolean" }
-            ]
```

Comment:
do we have any sort of replacement for this?  im guessing it's not really necessary since this has been disabled for so long anyways?

Reply:
Nope